File: Java/LeetCode/Count Prefixes of a Given String.java
// https://leetcode.com/problems/count-prefixes-of-a-given-string

class Solution {
  
    public int prefixCount(String[] words, String pref) {
        int count = 0;

        for (String word : words) {
            if (word.startsWith(pref)) {
                count++;
            }
        }

        return count;
    }
  
}

File: Kotlin/LeetCode/Count Prefixes of a Given String.kt
// https://leetcode.com/problems/count-prefixes-of-a-given-string

class Solution {
  
    fun countPrefixes(words: Array<String>, s: String): Int {
        var count = 0;
        
        for (word in words) {
            if (s.startsWith(word)) {
                count++
            }
        }

        return count;
    }
    
}
